velox/exec/AggregateCompanionAdapter.cpp

```diff
@@ -73,6 +73,14 @@ void AggregateCompanionFunctionBase::clearInternal() {
   fn_->clear();
 }
 
+void AggregateCompanionFunctionBase::initialize(
+    core::AggregationNode::Step step,
+    const std::vector<TypePtr>& rawInputType,
+    const facebook::velox::TypePtr& resultType,
+    const std::vector<VectorPtr>& constantInputs) {
+  fn_->initialize(step, rawInputType, resultType, constantInputs);
+}
```

Comment:
Let's move this method to every derived class since they need to pass different `step` to the main function's initialize() method.